en/views/helpers/html.rst

```diff
@@ -561,6 +561,66 @@ Once you have buffered javascript, you can output it as you would any other
     // In your layout
     echo $this->fetch('script');
 
+Creating Javascript Importmap
+-----------------------------
+
+.. php:method:: importmap(array $map, array $options = []): string
+
+Creates an `importmap` script tag for your JavaScript files::
+
+    // In the head tag of your layout
+    echo $this->Html->importmap([
+        'jquery' => 'jquery.js',
+        'wysiwyg' => '/editor/wysiwyg.js'
+    ]);
+
+Will output:
+
+.. code-block:: html
+
+    <script type="importmap">{
+        "imports": {
+            "jquery": "/js/jquery.js",
+            "wysiwyg": "/editor/wysiwyg.js"
+        }
+    }</script>
+
+Generating maps with imports, scopes and integrity::
+
+    echo $this->Html->importmap([
+        'imports' => [
+            'jquery' => 'jquery.js',
+            'wysiwyg' => '/editor/wysiwyg.js'
+        ],
+        'scopes' => [
+            'scoped/' => [
+                'foo' => 'inner/foo',
+            ],
+        ],
+        'integrity' => [
+            'jquery' => 'sha256-hash',
+        ],
+    ]);
+
+Will output:
+
+.. code-block:: html
+
+    <script type="importmap">{
+        "imports": {
+            "jquery": "/js/jquery.js",
+            "wysiwyg": "/editor/wysiwyg.js"
+        },
+        "scopes": {
+            "scoped/": {
+                "foo": "/js/inner/foo.js"
+            }
+        },
+        "integrity": {
+            "/js/jquery.js": "sha256-hash"
```

Comment:
I wonder if we should make this example more complete, or skip it? Having a broken example in the docs doesn't feel good, as the copy-paster is going to have an error. For this to work correctly, we need the `base64_encode(sha256)` of the file contents.

Reply:
I have updated the integrity example.